app/controllers/authentication_controller.rb

```diff
@@ -1,11 +1,21 @@
 class AuthenticationController < ApplicationController
   include DigitalIdentityHelper
 
+  DEFAULT_LEVEL_OF_AUTHENTICATION = "level0".freeze
```

Comment:
So this is just for a transition phase right?
Like eventually we'll remove this and just use the flag, but we're leaving this to ease transition?

Reply:
Yes, some more level-of-auth terminology goes away in [my branch implementing the DI thing](https://github.com/alphagov/account-api/compare/msw/simplify-levels-of-auth-2...msw/vtr)